File: VRP-Core/src/main/java/VRPCore/Cinematics/Cinematic.java
package VRPCore.Cinematics;

import VRPCore.Models.Vec5;
import org.bukkit.Location;

import java.util.ArrayList;

public class Cinematic {
    public String Name;
    public int Duration; // In Seconds
    public ArrayList<Vec5> KeyPoints;
    public String LinkedCinematic;
    public String[] Titles;
    public String[] Subtitles;
}

File: VRP-Core/VRP-Core/src/main/kotlin/Cinematics/Cinematic.kt
package VRPCore.Cinematics;

import VRPCore.Models.Vec5
import java.util.ArrayList

data class Cinematic(var Name: String?, var Duration: Int, var KeyPoints: ArrayList<Vec5>?, var LinkedCinematic: String, var Titles: Array<String>, var Subtitles: Array<String>) {

}